### PR TITLE
feat(suite): add Sepolia

### LINKED ISSUE #8360: Add Sepolia testnet to Trezor Suite
At the end of the year Goerli testnet will be deprecated and we need to replace it.
Sepolia testnet needs to be introduced in Trezor Suite.
https://sepolia1.trezor.io/

Do not remove Goerli yet.

Source: https://ethereum.org/en/developers/docs/networks/#ethereum-testnets
> Note: the Goerli testnet is deprecated and will be replaced by Holesovice in 2023. Please consider migrating your applications to Sepolia.

### PR DESCRIPTION
**Pull request by AdamSchinzel**
Add Sepolia testnet

## Description

Sepolia is now supported in testnet list.

## Related Issue

Resolve #8360 

## Screenshots:

Screenshot 2023-08-08 at 11 05 52 AM

Screenshot 2023-08-09 at 12 50 13 PM

### CODE REVIEW

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/SendCryptoInput/SendCryptoSelect/index.tsx [hunk 1]
```diff
@@ -48,7 +48,12 @@ const SendCryptoSelect = () => {
                         setValue(CRYPTO_INPUT, '');
                         setValue(FIAT_INPUT, '');
                         const token = selected.value;
-                        if (token === 'ETH' || token === 'TGOR' || token === 'ETC') {
+                        if (
+                            token === 'ETH' ||
+                            token === 'TSEP' ||
+                            token === 'TGOR' ||
+                            token === 'ETC'
```
Comment (dahaca) [inline packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/SendCryptoInput/SendCryptoSelect/index.tsx]: `['ETH', 'TSEP', 'TGOR', 'ETC'].includes(token)` and possibly use the network config to pull all `networkType: 'ethereum'` token names dynamically, especially considering it's used in multiple places.

File: suite-common/wallet-utils/src/validation.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { Account } from '@suite-common/wallet-types';
 
 const getNetworkType = (symbol: Account['symbol']) => {
     if (symbol === 'regtest') return symbol;
-    const testnets = ['test', 'txrp', 'tgor', 'tada'];
+    const testnets = ['test', 'txrp', 'tsep', 'tgor', 'tada'];
```
Comment (dahaca) [inline suite-common/wallet-utils/src/validation.ts]: Also possible to dynamically pull all testnets from the config.

Comment (AdamSchinzel) [reply suite-common/wallet-utils/src/validation.ts]: d159ece4d3309f70414e05e094d148f670899429

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/SendCryptoInput/SendCryptoSelect/index.tsx (hunk 1 above)
Comment (AdamSchinzel) [reply packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/SendCryptoInput/SendCryptoSelect/index.tsx]: d159ece4d3309f70414e05e094d148f670899429

**Review by tomasklim — APPROVED** [review]

**Pull request merged**